tests/test_EscCol.py

```diff
@@ -16,6 +17,40 @@ def test_basic_attributes(self):
         generation_time = 1.0 / (525600 / 20)
         assert round(abs(self.species.generation_time - generation_time), 7) == 0
 
```

Comment:
Terribly sorry, but one more thing: these next five tests are still bugging me. I think they're good, but they should be in `test_genome.py`, not in `test_EscCol.py`. The reason for this is that when someone else comes along to implement a new bacterial species, they'll look at EscCol and cpoy what's there, thinking they need to do all these tests, which are really tests of the basic machinery. Conversely, when someone goes and looks in test_genome.py to see how the basic machinery is being tested, they won't see it. So - how about just move these over there? They can be unchanged, just need a `species = stdpopsim.get_species("EscCol")` at the start?

Reply:
Don't worry, you are right, the best place for this is `test_genomes.py`.
I have moved the tests there now. It took me only a few minutes to do that. :)

Reply:
Thanks! I would have done it, but was traveling...